### ISSUE TITLE
Logitech G27 SDL mapping?

### ISSUE THREAD

**Issue by demonGeek** [virtual-programming/dirtshowdown-linux#26]
Does anyone have the SDL mapping info for a G27?

Thanks in advance.

**Comment by chrisaq** [comment]
Yes, also see https://github.com/chrisaq/lt-tools to set the correct wheel range.

```
030000006d0400009bc2000011010000,G27 Racing Wheel,platform:Linux,a:b6,b:b7,x:b10,y:b21,back:b8,start:b11,guide:b9,leftshoulder:b20,rightshoulder:b18,leftstick:b1,rightstick:b2,leftx:a0,lefttrigger:a2,righttrigger:a3,dpup:h0.1,dpleft:h0.8,dpdown:h0.4,dpright:h0.2,
```

I think the above was added to the game though.

**Comment by demonGeek** [comment]
Thanks Chris, much appreciated.

However, you were right, the game did already have a G27 SDL mapping but it's not working properly for me. The accelerator makes the car go in reverse and the brake makes it go forward.

I'm not really sure where to go from here...

**Comment by chrisaq** [comment]
@demonGeek Hi, no problem!
Your problem is in fact my fault!
I first posted my config in this thread: https://github.com/virtual-programming/dirtshowdown-linux/issues/11 , but I didn't expect it to be added to the game immediately, and of course I had made the mistake you're seeing. I then added the correct config(the one above) to the thread, but it seems like they didn't update the config.
So, if you replace the G27 line in this file:
`SteamApps/common/DiRT Showdown/SDLGamepad.config`
with the one above, then gas and brake should work as expected!
